pkg/storage/unified/apistore/store.go

```diff
@@ -70,6 +78,7 @@ func NewStorage(
 	getAttrsFunc storage.AttrFunc,
 	trigger storage.IndexerFuncs,
 	indexers *cache.Indexers,
+	largeObjectSupport bool,
```

Comment:
🤔  this is OK for this toy example, but not where we want to end up.

Ideally we could pass in a function that is defined in the apiserver init, but I am not sure how to best accomplish that.